app/controllers/Contributions.scala

```diff
@@ -106,6 +108,8 @@ class Contributions(paymentServices: PaymentServices, addToken: CSRFAddToken) ex
       .map(mobileRedirectUrl)
       .filter(_ => request.isIos)
 
+    info(s"thank you page displayed for paypal payment id: ${request.id}")
+    putThankYouPage()
```

Comment:
Can this and all other cloudwatch metrics calls be called `logThankYouPage`. I was a little confused about what this did from the name. Also the brackets are not required.